client/app/collections/filtered.coffee

```diff
@@ -88,6 +88,17 @@ module.exports = class FilteredCollection
         if query
             res = @underlying.filter @_filter
 
+            # Reduce to get max score and get a median limit, then excludes all
+            # results $lt it.
+            maxScore = res.reduce (memo, model) =>
```

Comment:
use `currentMax` instead of `memo` and `Math.max` for maximum ~~maxeadibility~~ readibility